realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -51,6 +52,12 @@
  * is required.
  * <p>
  * A RealmQuery cannot be passed between different threads.
+ * <p>
+ * Results are obtained fairly quickly most of the times. However, launching heavy queries from the UI thread may result
```

Comment:
Maybe it is just me but the term "fairly quickly" feels like a cop out. Either just say "Quickly" or remove the line completely IMO